docs/cli.rst

```diff
@@ -48,6 +51,12 @@ Or with a filename::
     export FLASK_APP=/path/to/hello.py
     flask run
 
+If you use Flask 0.13 or later you can also set the environment variables
+in a ``.env`` file.  For instance like this::
+
+    FLASK_APP=/path/to/hello.py
+    FLASK_DEBUG=1
+
```

Comment:
I have mixed feelings regarding including `FLASK_APP` in this example. The typical usage of a `.env` file is for configuration items that change between installations (i.e. local machine vs production server). However, the Flask app instance is fixed. I understand why you would want to put it here, as it annoys me to no end to have to set `FLASK_APP` to be able to use the Flask cli, but I'm thinking people will be inclined to add the `.env` file to source control only so that they can forget about `FLASK_APP`, and if they also store sensitive information such as email server passwords, AWS creds, etc. in this file (as that is one of the most interesting uses of a `.env` file IMHO) then we have a problem.